modules/core/src/main/java/org/apache/ignite/internal/managers/systemview/walker/AtomicReferenceViewWalker.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2019 GridGain Systems, Inc. and Contributors.
```

Comment:
what reason? here and below

Reply:
it`s auto generated code, ok i revert it.